### PR TITLE
Safari#120

### LINKED ISSUE #120: Safari desktop and all browsers on iOS will not play videos
Also see this [companion issue on the front end](https://github.com/BYU-ODH/yvideo-client/issues/464).

The issue appears to be a problem with any browser using the WebKit engine (Safari on all devices, and all browsers on iOS). Allowing students to watch on their iOS devices would be very helpful. There are very many descriptions of the Safari video problem online:

https://blog.logrocket.com/streaming-video-in-safari/
https://stackoverflow.com/questions/27712778/video-plays-in-other-browsers-but-not-safari

These browsers require `Partial-Content` (status code `206`), rather than simple file transfer. We have implemented `Partial Content`, and it works great on Chrome, so it is puzzling why this is not working in Safari.

One major obstacle for local development for Safari is that it requires an SSL connection over https. We have not been able to get the configurations right for this to work, so development has consisted of making changes, merging them into `develop` to deploy to the dev server (!!!), and then accessing yvideodev.byu.edu using Safari. Very slow and muddies the git flow. The mechanics of developing for Safari have really gotten in the way.

FWIW, it is possible to put a macOS VM on Linux: https://github.com/myspaghetti/macos-virtualbox.

### PR DESCRIPTION
**Pull request by WorldsEndless**
closes #120. 

Applies content-type of `video/mp4` to everything. Are there other types we serve? (This is tricky because no file extensions are provided to the service handler)

### CODE REVIEW

**Comment by WorldsEndless** [comment]
This one is awaiting testing on apple devices. It should be merge-able, though, because the worst it can do is leave those who are already broken still broken.

File: src/clj/y_video_back/middleware.clj [hunk 1]
```diff
@@ -238,13 +240,14 @@
 
 (defn wrap-api-post [handler]
   (-> handler
+      ;;ring-content/wrap-content-type
```
Comment (reynoldsnlp) [inline src/clj/y_video_back/middleware.clj:243]: Delete this comment?

File: src/clj/y_video_back/routes/service_handlers/handlers/media_handlers.clj [hunk 2]
```diff
@@ -6,10 +6,15 @@
...
-   [y-video-back.log :as log-ut]))
+   [ring.util.response :as rs :refer [file-response]]
+   [ring.mock.request :as mr]
+   [y-video-back.log :as log-ut]
+   [taoensso.timbre :as log]))
 
-; TODO - check if user has permission to stream requested file
+                                        ; TODO - check if user has permission to stream requested file
+
+(defn extension [s]
```
Comment (reynoldsnlp) [inline src/clj/y_video_back/routes/service_handlers/handlers/media_handlers.clj:16]: I don't know if using short variables like `s` (for "string"?) is conventional in Clojure, but I think more verbose variable names are almost always readable/maintainable. The only exception I make for myself is when it is strongly suggests by context, e.g. `for s in strings`.

**Review by reynoldsnlp — CHANGES_REQUESTED** [review]
Tiny edits requested.

**Review by reynoldsnlp — APPROVED** [review]
lgtm

**Pull request merged**